spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -138,6 +153,9 @@ data class SpdxLicenseRefExpression(
 ) : SpdxExpression() {
     override fun spdxLicenses() = enumSetOf<SpdxLicense>()
 
+    // TODO: Think about whether we should also accept "DocumentRef-" here, or model those as a separate class.
```

Comment:
In the specs both are grouped in `license-ref = ["DocumentRef-"1*(idstring)":"]"LicenseRef-"1*(idstring)` and this what the ANTLR grammar also allows: `LICENSEREF : ('DocumentRef-' | 'LicenseRef-') IDSTRING ;`. So I would say we should accept it here and we don't need a separate class.

Reply:
I just realized the ANTLR grammar is wrong, DocumentRef always has to be a prefix of a LicenseRef like in "DocumentRef-spdx-tool-1.2:LicenseRef-MIT-Style-2". Do you want to fix this as part of this PR?

Reply:
See https://github.com/heremaps/oss-review-toolkit/pull/1202.